### PR TITLE
Modified node app

### PR DESCRIPTION
**Pull request by codewdhruv**

### CODE REVIEW

File: .harness/nodejsstarterpipeline.yaml [hunk 1]
```diff
@@ -0,0 +1,96 @@
+pipeline:
+  name: nodejs-starter-pipeline
+  identifier: nodejsstarterpipeline
+  projectIdentifier: DhrubaCI
```
Comment (Ompragash) [inline .harness/nodejsstarterpipeline.yaml]: Change the Project Identifier to a generic name

File: .harness/nodejsstarterpipeline.yaml [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+                    shell: Sh
+                    command: |-
+                      cd /nodejs-pipeline-samples
+                      node index.js
+              - step:
+                  type: Run
+                  name: test connection to server
+                  identifier: test_connection_to_server
+                  spec:
+                    connectorRef: DhrubajyotiDocker
```
Comment (Ompragash) [inline .harness/nodejsstarterpipeline.yaml]: Change the connector ref name

File: .harness/inputset.yaml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+        type: "CI"
+        spec:
+          execution:
+            steps:
+            - step:
+                identifier: "build_and_push_image_to_docker"
+                type: "BuildAndPushDockerRegistry"
+                spec:
+                  connectorRef: "DhrubajyotiDocker"
+                  repo: "dhrubajyotichakraborty/nodejshelloworld"
```
Comment (Ompragash) [inline .harness/inputset.yaml]: Do the users need to mention the forked repo? If yes, keep it more generic like `GITUSERNAME/FORKEDREPO`

**Review by Ompragash — CHANGES_REQUESTED** [review]

File: .harness/nodejsstarterpipeline.yaml (hunk 1 above)
Comment (codewdhruv) [reply .harness/nodejsstarterpipeline.yaml]: removed the identifiers as they'll be unique to the user

File: .harness/inputset.yaml (hunk 3 above)
Comment (codewdhruv) [reply .harness/inputset.yaml]: DONE

**Review by Ompragash — APPROVED** [review]
Thanks!

**Pull request merged**